### ISSUE TITLE
Accuracy

### ISSUE THREAD

**Issue by LKELN** [bytedance/R2Former#11]
Thanks for your work! But I still can't get the same accuracy as you on train reranking , even if the environment and train_reranking.sh parameters I use are the same as you provided, can you tell me why?
[info.log](https://github.com/bytedance/R2Former/files/13166479/info.log)

**Comment by Jeff-Zilence** [comment]
Since you are using 2GPUs, the learning needs to be tuned. Also you need to do the finetuning to match our final results. See the second command in train_reranking.sh.

**Comment by LKELN** [comment]
Even though I'm using two GPUs, all my training parameters are the same as yours, and as you said, it should be around 88.4 without fine-tuning.And When I train the end-to-end model, the training accuracy is as you said. This accuracy fluctuates too much, can you fix this problem?

**Comment by szhu-bytedance** [comment]
The batch size per GPU will have impact on the performance for all distributed training pipeline. I am not able to debug for you given the limited information.
